mule-extensions-soap-api/src/main/java/org/mule/runtime/extension/api/soap/SoapOutputPayload.java

```diff
@@ -49,4 +57,26 @@ public SoapOutputPayload(TypedValue<InputStream> body,
   public Map<String, TypedValue<String>> getHeaders() {
     return headers;
   }
+
+  @Override
+  public String toString() {
+    try {
+      String hs = headers.values().stream().map(v -> "\"" + v.getValue() + "\"").collect(joining(",\n  "));
+      String as = String.join(", ", attachments.keySet());
+      String bodyString = IOUtils.toString(body.getValue(), getBodyCharset());
+      return "{\n" +
+          "body:" + bodyString + ",\n" +
+          "headers: [" + hs + "]" + ",\n" +
+          "attachments: [" + as + "]" + "\n" +
+          "}";
+    } catch (IOException e) {
+      throw new RuntimeException("Cannot transform body to string");
```

Comment:
Is not a good idea to fail on a to String, return a different string saying that the String could not be generated, or something like this.